internal/cmd/commands/dev/dev.go

```diff
@@ -427,14 +436,18 @@ func (c *Command) Run(args []string) int {
 	c.Config.Plugins.ExecutionDir = c.flagPluginExecutionDir
 	c.Config.Worker.AuthStoragePath = c.flagWorkerAuthStorageDir
 	if c.flagIdSuffix != "" {
```

Comment:
We're not validating the secondary id suffix if the first one isn't set.

Reply:
Good catch, completely missed this top-level if statement!

Reply:
Fixed with [e26ee9b](https://github.com/hashicorp/boundary/pull/2738/commits/e26ee9bde08e04a2e66ef5a981a0e1b895e29e5a)